yarn.lock

```diff
@@ -1221,7 +1221,7 @@ class-utils@^0.3.5:
     isobject "^3.0.0"
     static-extend "^0.1.1"
 
-classnames@^2.2.5:
+classnames@^2.2.6:
```

Comment:
Should we be updating `yarn.lock` in this diff? We didn't add any new packages?

Reply:
I just did `yarn install` and it got updated :/

Reply:
Ok. I am surprised these updates were not caught when we published 1.2.0. But, hey, move fast and ship it 😄